apps/core0/bootstrap/bootstrap.go

```diff
@@ -137,15 +137,23 @@ func (b *Bootstrap) setupNetworking() error {
 		return fmt.Errorf("failed to get network interfaces: %s", err)
 	}
 
+	ignore, _ := options.Options.Kernel.Get("noautonic")
```

Comment:
Seems you're masking the error here, this is very bad practice, please check the error